content/_guides/sasl/emacs-erc.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Configuring SASL for Emacs ERC
+category: sasl
+credits: androclus
+---
+
+[ERC](https://www.emacswiki.org/emacs/ERC) is an IRC client which comes as a package for Emacs.
+
+## A. Set up ERC IRC client
+
+If you have ERC already installed and running on your emacs system, then skip to part B.
```

Comment:
Without numbered headings you could just link to them instead

```suggestion
If you have ERC already installed and running on your emacs system, then skip to
[registration](#registration).
```

Reply:
Okay, I've redone these. I am assuming that being on github.io, these ## sections will get automatically anchored to #user-content-*, as toto_tico's comment at https://stackoverflow.com/questions/5319754/cross-reference-named-anchor-in-markdown seems to indicate. Otherwise, let me know another way you would like me to mark down the cross-reference anchor.

Reply:
I believe the `user-content-` prefix only applies to content displayed on github.com, but for the site it's processed with Jekyll it doesn't get any prefix, the links you added looks correct if the prefix is removed

Reply:
Okay, these removed